src/Joomlatools/Console/Command/Site/Listing.php

```diff
@@ -91,12 +99,26 @@ protected function execute(InputInterface $input, OutputInterface $output)
             }
         }
 
-        $i = 1;
-        foreach ($sites as $site) {
+        if (!in_array($input->getOption('format'), ['txt', 'json'])) {
+            throw new \InvalidArgumentException(sprintf('Unsupported format "%s".', $input->getOption('format')));
+        }
+
+        if ($input->getOption('format') == 'json') {
+            $result = new \stdClass();
+            $result->command = $input->getArgument('command');
+            $result->sites = [];
+            foreach ($sites as $site) {
+                $result->sites[] = (object) $site;
+            }
+            $output->writeln(json_encode($result));
+        } else if ($input->getOption('format') == 'txt') {
```

Comment:
We should simply make this an `if/else` statement because `txt` is the default output anyway. Can you remove the ìf ($input->getOption('format') == 'txt')` part?